migrations.js

```diff
@@ -79,15 +121,18 @@ var runMigrations = function(meta, migrations, callback) {
 
 module.exports = {
   run: function(callback) {
-    getMeta(function(err, meta) {
+    module.exports.get(function(err, migrations) {
       if (err) {
         return callback(err);
       }
-      module.exports.get(function(err, migrations) {
+      if (!migrations.length) {
```

Comment:
That's a good point. We have hardcoded at least one migration! I got a bit overzealous with edge cases. I'll remove it.